Don't error in DogtagCertsConnectivityCheck with external CAs

rcritten opened the pull request:
The serial number of externally-signed CA's were being checked using ipa-cert-find which invariably would return a not found error.

Instead return SUCCESS to effectively skip the check.

Fixes: https://github.com/freeipa/freeipa-healthcheck/issues/285

flo-renaud reviewed (commented):
Hi @rcritten 
if IPA is installed with embedded CA, the connectivity check can use the RA cert in all the cases (self-signed or externally signed CA), as this cert always exists and can be found with PKI API.

rcritten commented:
Excellent observation! Implemented.

flo-renaud reviewed (commented):
Hi @rcritten 
thanks for the update. Minor nitpicks, not related to this patch but that could be fixed with it:
- line 141, the message is missing a closing }
- lines 126, 132 and 138, we could replace the key 'cert_show_1' with 'cert_show_ra'

I tested the following scenarios with an externally-signed CA:
- missing RA cert file
- RA cert file contains a cert inconsistent with what is stored in LDAP
- pki stopped
All the above issues were properly reported.

rcritten commented:
> thanks for the update. Minor nitpicks, not related to this patch but that could be fixed with it:
> 
>     * line 141, the message is missing a closing }

Fixed.

>     * lines 126, 132 and 138, we could replace the key 'cert_show_1' with 'cert_show_ra'

Done. I modified the test expectations as well, including the typical serial number for the RA (it doesn't really matter in the end but non-zero is more obvious)

flo-renaud reviewed (approved):
@rcritten 
thanks for the update, works for me. I tested the following scenarios with an externally-signed CA:

- missing RA cert file
- RA cert file contains a cert inconsistent with what is stored in LDAP
- pki stopped

All the above issues were properly reported.
